src/controllers/CustomerInfoController.java

```diff
@@ -42,46 +65,60 @@ public void handle(MouseEvent event) {
                 String zipCode = zipCodeField.getText();
                 String state = stateField.getText();
                 String phoneNumber = phoneNumberField.getText();
-
                 CustomerModel customerModel = new CustomerModel(firstName, lastName, addressOne, addressTwo, phoneNumber, state, zipCode, city);
                 customerProfile.put(customerModel.getLastName(), customerModel);
-                getCustomerInfo(customerProfile);
-//                movingToNextScreen(customerProfile);
+                applicationContext.setCurrentCustomer(customerModel);
+                mainInterfaceStage.stage(placeOrderButton);
```

Comment:
transition to a single service responsible for taking in a button and a location to switch to.